elastic/integrations
# [PFsense] Add PFsense Integration

## What does this PR do?

Add PFsense Integration.  Currently parses Firewall (filterlog), Open VPN (openvpn), IPSEC (charon), DHCP (dhcpd), DNS (unbound), HAProxy (haproxy) logs

## Checklist


## Author's Checklist


## How to test this PR locally

```
cd integrations/packages/pfsense
elastic-package check && elastic-package stack down && elastic-package build && elastic-package stack up -d -v -p default && eval "$(elastic-package stack shellinit)" && elastic-package test  -v
```

## Related issues

- Closes #1206 

## Screenshots
